extensions/wikia/ThemeDesigner/ThemeDesignerHelper.class.php

```diff
@@ -55,7 +55,9 @@ public static function isValidColor( $sColor ) {
 		// Last chance: array is not proper hash so maybe it is predefined color name
 		$isColorName = in_array( strtolower( $sColor ), static::COLORS );
 
-		Wikia::log( __METHOD__, "JKU", "[{$sColor}] is not a valid color name" );
+		if ( !$isColorName ) {
+			Wikia::log( __METHOD__, "JKU", "[{$sColor}] is not a valid color name" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace or remove this deprecated Wikia::log logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)